Examples/CodePushDemoApp/android/app/react.gradle

```diff
@@ -0,0 +1,75 @@
+def config = project.hasProperty("react") ? project.react : [];
+
+def bundleAssetName = config.bundleAssetName ?: "index.android.bundle"
+def entryFile = config.entryFile ?: "index.android.js"
+
+// because elvis operator
```

Comment:
This was also generated code from running the `react-native upgrade` command

https://github.com/facebook/react-native/blob/4fc3e26cc8794e0d40d45abd23d065dd18de4e22/local-cli/generator-android/templates/src/app/react.gradle